README.md

```diff
@@ -1,5 +1,18 @@
-# license-list-XML
-This is a workspace for developing and implementing a new XML source format for the SPDX License List. This source format is (or will be) used to generate the authoritative file formats for licenses on the SPDX License List. These authoritative [target/destination formats](https://github.com/spdx/license-list-data) will remain the same for the time being.
+# SPDX License List 
+The [SPDX License List](https://spdx.org/licenses/) is a list of commonly found licenses and exceptions used in free and open source and other collaborative software or documentation. The purpose of the SPDX License List is to enable easy and efficient identification of such licenses and exceptions in an SPDX document, in source files or elsewhere. The SPDX License List includes a standardized short identifier, full name, vetted license text including matching guidelines markup as appropriate, and a canonical permanent URL for each license and exception. 
+
+# This Repository
+This repository contains the XML source and schema files used to generate the authoritative, supported SPDX list file formats, including the web pages you see at [spdx.org/licenses](https://spdx.org/licenses/) and other generated data formats found in the [SPDX license-list-data repository]. 
+
+## This Repository Is Unstable
+Please note that the format for this repository is internal to the SPDX legal team and is subject to change. Specifically, the XML format for the SPDX source files here is currently unstable, so any direct consumers of _this_ repository's source files should expect occasional, backwards-incompatible changes. 
+
+In contrast, the output files in the [SPDX license-list-data repository](https://github.com/spdx/license-list-data) are stable and well-supported, and make the License List available in RDFa, HTML, text, and JSON formats. You can use [SPDX tools](https://github.com/spdx/tools) (or create your own) to consume the supported formats of the license list.
```

Comment:
With a reference-style link to license-list-data (as I suggest above), this entry can reuse the same reference with:

```markdown
… the [SPDX license-list-data repository][license-list-data] are…
```